internal/server/kong/ws/config_service.go

```diff
@@ -73,6 +74,16 @@ func (c *configService) PingCP(
 			Value: "wrpc",
 		},
 	)
+	dpPingCounter.Inc(
```

Comment:
Can we move these new metrics to another commit, as they're not related to these changes?

Reply:
done [630b8de](https://github.com/Kong/koko/pull/364/commits/630b8debe9b656d21379d237ac8f3989297d1bc4)

Reply:
Saw that these were removed, do we want to just implement it in another PR then?

Reply:
Sorry, I misunderstood "another commit" to mean "another PR". Let's go with another PR either way since these seem to replace the existing metrics, but the PR when handed off to me was duplicating rather than replacing. I'll handle the replacement in isolation.

Reply:
I'm working on a follow-up PR that implements various dp <-> cp metrics, I'll add this removed commit into those batch of changes. Branch here: https://github.com/Kong/koko/compare/feat/metrics-v2...feat/dp-metrics

Reply:
Work with me, that sounds reasonable anyway.